eng/tools/specs-model/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@azure-tools/specs-model",
+  "private": true,
+  "type": "module",
+  "main": "dist/src/index.js",
+  "engines": {
+    "node": ">= 18.0.0"
+  },
+  "scripts": {
+    "build": "tsc",
+    "postinstall": "npm run build",
+    "test": "vitest",
+    "test:ci": "vitest run --coverage --reporter=verbose",
+    "lint": "eslint . -c eslint.config.js --report-unused-disable-directives --max-warnings 0",
+    "lint:fix": "eslint . -c eslint.config.js --fix",
+    "prettier": "prettier . --check",
+    "prettier:debug": "prettier . --check ---log-level debug",
+    "prettier:write": "prettier . --write"
+  },
+  "bin": {
+    "get-specs-model": "cmd/get-specs-model.js"
+  },
+  "dependencies": {
+    "lodash": "^4.17.21"
+  },
+  "devDependencies": {
+    "@eslint/js": "^9.7.0",
+    "@tsconfig/strictest": "^2.0.5",
+    "@types/eslint__js": "^8.42.3",
+    "@types/lodash": "^4.17.6",
+    "@types/node": "^20.14.10",
+    "@vitest/coverage-v8": "^2.0.2",
+    "eslint": "^9.7.0",
+    "eslint-plugin-unicorn": "^54.0.0",
+    "typescript": "~5.5.3",
```

Comment:
FYI some of the dependencies I use here are newest, newer than in `suppressions`. Like `typescript` or `vitest`. Not sure what is our approach for keeping them in sync.

Reply:
As much as possible, I have been aligning versions in `azure-rest-api-specs` with the `microsoft/typespec` repo.  Additionally, all tools under eng should be aligned.

Reply:
I aligned versions in `package.json` files for specs-model and suppressions, by downgrading specs-model versions to match.

I also reordered the keys a bit, to be in following order:

name
private
main
bin
scripts
engines
dependencies
devDependencies

The reasoning here is that `bin` and `scripts` should be near the top, as these are the entry-points.